renderer/main.js

```diff
@@ -231,13 +231,13 @@ const dispatchHandlers = {
   'forward': () => state.location.forward(),
 
   // Controlling the window
-  'setDimensions': (dimensions) => setDimensions(dimensions),
+  'setDimensions': setDimensions,
```

Comment:
This was to be consistent with `backToList` and `escapeBack`. Do we want this? If so do we also want to call state functions directly?